crDroidAndroid-14.0-20240113-avicii-v10.1.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "PSavarMattas",
+			"oem": "OnePlus",
+			"device": "OnePlus Nord",
+			"filename": "crDroidAndroid-14.0-20240113-avicii-v10.1.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/avicii/10.x/crDroidAndroid-14.0-20240113-avicii-v10.1.zip/download",
+			"timestamp": 1705109739,
+			"md5": "51c7c9b3ebd8f5bad8da664b63e17fe8",
+			"sha256": "ff5372525f0fa9e888482f96afaaed8caad7f2dcd9e83562a7c5ec69da7c32f6",
+			"size": 1195604180,
+			"version": "10.1",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-official-14-oneplus-nord-avicii-crdroid-v10-x.4650600/",
+			"gapps": "",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "",
+			"paypal": "https://www.paypal.me/psmattas610",
+			"telegram": "https://t.me/psmbuilds",
+			"dt": "https://github.com/crdroidandroid/android_device_oneplus_avicii",
```

Comment:
permissive is not valid
https://github.com/crdroidandroid/android_device_oneplus_avicii/blob/14.0/BoardConfig.mk#L123

Reply:
Sir the build is not permissive just forgot to push the latest changes to the DT. 

the commit is now made: https://github.com/crdroidandroid/android_device_oneplus_avicii/commit/09e555742e6064e70e61d0fc4aa3fcbba0c620bc